src/components/accessories/patientTriage/patientTriageForm/types.ts

```diff
@@ -0,0 +1,28 @@
+import { TFields } from "../../../../libraries/formDataHandling/types";
+
+interface IOwnProps {
```

Comment:
Now that we have more than one form, we could standardize this form component props interface and use extensions when we need specific stuff. What do you think?

Reply:
I open a separate branch in order to fix this.